test-clients/src/main/java/com/hedera/services/bdd/suites/perf/PerfUtilOps.java

```diff
@@ -77,4 +77,16 @@ public static HapiTxnOp tokenOpsEnablement() {
 						entry("tokenDissociateFromAccount", "0-*")
 				));
 	}
+
+	public static HapiTxnOp scheduleOpsEnablement() {
+		return fileUpdate(API_PERMISSIONS)
+				.fee(9_999_999_999L)
```

Comment:
nit: We can use `HapiApiSuite.A_HUNDRED_HBARS` to remove yet another magic number/constant here.

Reply:
Fixed. Thanks @qnswirlds !